README.md

```diff
@@ -1 +1,366 @@
+| Y | Confirm arrival point and timeslot page | confirm-arrival-point | confirm user selection for arrival point and timeslot | Ok |
+| Y | Confirm visitor details page | get-confirmed-visitor-details | load user provided visitor data | stored visitor data list |
+| Y | Confirm visitor details page | store-visitors | user provided visitor data | Ok |
+| Y | Confirm transport types page | get-transport-types-selection | load stored transport types | selected transport type |
+| Y | Confirm transport types page | store-transport-types | store user transport type information | Ok |
+| Y | Confirm transport types page | get-point-of-origin | point of origin + selection flag | list of origin points |
+| Y | Summary page | get-reservation | load in progress reservation details | in progress reservation details |
+| Y | Summary page | finalize-reservation | make reservation final | Ok |
+| N | Welcome page | get-daily-available-places | get places without login  | date + yes and no if there are at least 6 places available |
+| Y | Update arrival point page | update-arrival-point | store arrival point changes | Ok |
+| Y | Update arrival point page | get-reservation-arrival-data | get reservation arrival point data + available timeslots | timeslot change possibilities |
+| Y | Update transport types page | get-point-of-origin | point of origin + selection flag | list of origin points | 
+| Y | Update arrival point page | update-transport-types | store user transport type changes | Ok |
+| Y | Dashboard page | resend-reservation-confirmation | send again confirmation mail  | Ok |
+| Y | Dashboard page | get-update-reservation-token | send again  | JWT token with required reservation id |
+| Y | Visitrs data update page | get-visitor-details-for-update | load visitors data for update | visitors data in array |
+| Y | Visitrs data update page | update-visitors | store modified visitors data | Ok |
+| N | - | event-registration | test api for post message test | test values |
+| N | - | request-passcode-for-testing | request passcode for automated testing. Works only for "dev" and "qa" environment | passcode |
+
+## Workflows
+
+This applicaiton is designed to work as individual API calls, which means they are not relying on other API calls. The [front-end user interface](https://github.com/cerndb/open-days-registration-application-front-end) creates an environment and ensures calling the APIs in order. As a result of the user interface walkthrough a reservation can be created, updated and cancelled.
+
+### login workflow
+
+Using the front-end user interface you request a passcode and with that passcode you login. The front-end will receive a token generated by the back-end and to each call the back-end server will receive this token. If the token is expired or incorrect the back-end server will send back an error message. 
+
+### reservation workflow
```

Comment:
I would use upper case at the beginning for the headings, for instance ### Reservation workflow

Reply:
OK, I have applied the changes on the other headers as well